qa/Rakefile

```diff
@@ -9,19 +9,42 @@ platforms = PlatformConfig.new
 task :spec    => 'spec:all'
 task :default => :spec
 
-namespace :test do
-  desc "Generate a valid ssh-config"
-  task :ssh_config do
-    require "json"
-    raw_ssh_config    = LogStash::VagrantHelpers.fetch_config.stdout.split("\n");
-    parsed_ssh_config = LogStash::VagrantHelpers.parse(raw_ssh_config)
-    File.write(".vm_ssh_config", parsed_ssh_config.to_json)
-  end
+namespace :qa do
+
+  namespace :vm do
+
+    desc "Generate a valid ssh-config"
+    task :ssh_config do
+      require "json"
+      raw_ssh_config    = LogStash::VagrantHelpers.fetch_config.stdout.split("\n");
+      parsed_ssh_config = LogStash::VagrantHelpers.parse(raw_ssh_config)
+      File.write(".vm_ssh_config", parsed_ssh_config.to_json)
+    end
+
+    desc "Bootstrap all the VM's used for this tests"
+    task :setup, :platform do |t, args|
+      config   = PlatformConfig.new
+      machines = config.select_names_for(args[:platform])
+
+      message  = "bootstraping all VM's defined in acceptance/Vagrantfile"
+      message  = "#{message} for #{args[:platform]}: #{machines}" if !args[:platform].nil?
+      puts message
+
+      LogStash::VagrantHelpers.destroy   machines
```

Comment:
many spaces between destroy and machine and should we use parentheses for theses methods calls?